### PR TITLE
Login test 2

### PR DESCRIPTION
**Pull request by cleborys**

### CODE REVIEW

File: login_test.py [hunk 1]
```diff
@@ -30,4 +30,40 @@ def test_unverifiedUserFirstLogin(driver, unverified_user_via_medlemshjaelp):
...
 
+# Test specification number 2
+def test_unverifiedUserFirstLoginDuplicateTab(driver, unverified_user_via_medlemshjaelp):
+    user = unverified_user_via_medlemshjaelp
+
+    navigate_to_page("login", driver)
+    first_tab = driver.current_window_handle
+
+    navigate_to_page("login", driver, new_tab=True)
+    WebDriverWait(driver, 10).until(EC.number_of_windows_to_be(2))
```
Comment (FjolleJagt) [inline login_test.py]: Can we move this line inside the `navigate_to_page` function? (There's hopefully some way to detect how many tabs are open already. I assume that this isn't caught by the WebDriverWait already in `navigate_to_page`?)

File: login_test.py [hunk 2]
```diff
@@ -30,4 +30,40 @@ def test_unverifiedUserFirstLogin(driver, unverified_user_via_medlemshjaelp):
...
+    second_tab = [x for x in driver.window_handles if x != first_tab][0]
```
Comment (FjolleJagt) [inline login_test.py]: And `navigate_to_page` could return the current handle automatically.

File: login_test.py [hunk 3]
```diff
@@ -30,4 +30,40 @@ def test_unverifiedUserFirstLogin(driver, unverified_user_via_medlemshjaelp):
...
+    driver.switch_to.window(second_tab)
+    navigate_to_page("bekraeft_konto", driver)
+    assert_current_page_is("bekraeft_konto", driver)
+    fill_form_field(token, driver, form_id="input_verification_code")
+    submit_form(driver)
+    assert_current_page_is("opret_password", driver)
+    fill_form_field(user["password"], driver, form_id="input_new_password")
+    fill_form_field(user["password"], driver, form_id="input_confirm_password")
+    submit_form(driver)
+    assert_current_page_is("kvittering", driver)
```
Comment (FjolleJagt) [inline login_test.py:40]: Similarly to the last test, some of these can be replaced by higher-level abstractions (e.g. "create password"), some of which already exist from Login test 1

**Review by FjolleJagt — COMMENTED** [review]
Looks good, though I would prefer to re-review after the test has been rewritten with higher-level concepts

File: login_test.py (hunk 2 above)
Comment (cleborys) [reply login_test.py]: Nice!

**Review by FjolleJagt — APPROVED** [review]
Looks good to me!

**Pull request merged**